assets/js/components/DatabaseDetails/DatabaseDetails.jsx

```diff
@@ -14,11 +16,14 @@ function DatabaseDetails() {
   }
 
   return (
-    <GenericSystemDetails
-      title="HANA Database Details"
-      type={DATABASE_TYPE}
-      system={database}
-    />
+    <>
+      <BackButton url="/databases">Back to HANA Databases</BackButton>
+      <GenericSystemDetails
+        title="HANA Database Details"
```

Comment:
In hosts and cluster view the `details` word is in small case, but the figma mock has this word in capital letter